src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverAdapter.java

```diff
@@ -5,14 +5,17 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+
 /**
- * driver adapter to drawer with several bugs.
+ * driver adapter to drawer
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+
+public class DriverAdapter implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.